sample.env

```diff
@@ -54,7 +54,7 @@ NGINX_PROXY_PORT=80
 ## Docker image versions
 ## 
 STACKS_BLOCKCHAIN_VERSION=2.5.0.0.3
-STACKS_BLOCKCHAIN_API_VERSION=7.10.1
+STACKS_BLOCKCHAIN_API_VERSION=7.10.0
```

Comment:
maybe a good idea to use the latest [7.11.0](https://github.com/hirosystems/stacks-blockchain-api/releases/tag/v7.11.0) here?

Reply:
That version was released after my PR. Now its updated to 7.11.0.